src/main/java/org/broadinstitute/consent/http/service/SummaryService.java

```diff
@@ -295,23 +293,24 @@ public File describeDataAccessRequestSummaryDetail() {
                                     summaryWriter.write("-" + SEPARATOR);
                                     summaryWriter.write("-" + SEPARATOR);
                                 }
-                                summaryWriter.write( dar.get(DarConstants.INVESTIGATOR)  + SEPARATOR);
-                                summaryWriter.write( dar.get(DarConstants.PROJECT_TITLE)  + SEPARATOR);
-                                List<Integer> dataSetIds = DarUtil.getIntegerList(dar, DarConstants.DATASET_ID);
+                                User u = userDAO.findUserById(dar.getUserId());
+                                summaryWriter.write( u.getDisplayName()  + SEPARATOR);
+                                summaryWriter.write( dar.getData().getProjectTitle() + SEPARATOR);
+                                List<Integer> dataSetIds =  dar.getData().getDatasetIds();
```

Comment:
nit, this isn't necessary since you have `List<Integer> datasetId = dar.getData().getDatasetIds();` on line 273. The original code was making the same mistake.